rust/src/lightning.rs

```diff
@@ -1001,6 +1004,7 @@ async fn handle_ldk_events(
                     Flow::Inbound,
                     HTLCStatus::Succeeded,
                     MillisatAmount(Some(*amount_msat)),
+                    None,
```

Comment:
🙃 It's weird to have to deal with this here because the payment has already succeeded.

Reply:
there's definitely potential for better types and better API here.
I believe the awkwardness is that you might receive a payment without creating an invoice beforehand, hence you only learn about it when you get it.